frontend/src/lang/modules/zh.ts

```diff
@@ -1666,9 +1665,7 @@ const message = {
         systemLogHelper: '系统日志信息、容器构建或镜像拉取等日志信息以及计划任务中产生的日志文件',
         dockerLog: '容器操作日志文件',
         taskLog: '计划任务执行日志文件',
-        shell: 'Shell 脚本计划任务',
-        containerShell: '容器内执行 Shell 脚本计划任务',
-        curl: 'CURL 计划任务',
+        containerShell: '容器内 Shell 脚本',
 
         containerTrash: '容器垃圾',
         images: '镜像',
```

Comment:
The provided code changes appear to be small edits without significant impact on the overall structure or functionality. The primary change is replacing `snapshot` with `snapshotTmp`, which likely represents temporary files used during snapshot upload.

**Irregularities/Issues**: None spotted.

**Optimization Suggestions**: 
- Ensure that file names and identifiers remain consistent throughout the project, especially when they refer to similar concepts (e.g., renaming `snapshot` to another variable name consistently).
- Double-check if any logic depends on the presence of these files, ensuring that the new naming does not inadvertently affect behavior.
- Verify that there are no existing references to the old `snapshot` path that would need updating along with the rename.

Overall, this change should have a minimal effect but good practices suggest consistency in naming conventions across the codebase.